src/SFA.DAS.EmployerFinance/Infrastructure/OuterApiResponses/Transfers/GetIndexResponse.cs

```diff
@@ -4,5 +4,6 @@ public class GetIndexResponse
     {
         public int PledgesCount { get; set; }
         public int ApplicationsCount { get; set; }
+        public int ActivePledgesTotalAmount { get; set; }
```

Comment:
Sorry, does this field needs to be decimal as well or this one is okay? -->> public int ActivePledgesTotalAmount